Sessions/5a-word2vec-exploring.ipynb

```diff
@@ -0,0 +1,771 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# KeyedVectors allow word vectors to be accessed as if from a dictionary:\n",
+    "print(wiki_vectors[\"pear\"])"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Option 2: loading a model from elsewhere"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Word2Vec models are trained and shared all the time, and most of them are not available through the Gensim downloader. In fact, not all of them are easy to find, sometimes it requires doing a bit of archeology to find whether something you would find useful already exists and has been shared. For example, it's common for researchers to train their own models and upload them to Zenodo or another research repository. Or different institutions may have their own repositories.\n",
+    "\n",
+    "For example, you can find word vectors in many different languages [here](https://fasttext.cc/docs/en/crawl-vectors.html#models):\n",
+    "> E. Grave, P. Bojanowski, P. Gupta, A. Joulin, T. Mikolov, Learning Word Vectors for 157 Languages. I Proceedings of the International Conference on Language Resources and Evaluation, 2018."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Pre-trained models often come as:\n",
+    "* either a vectors-only file (i.e. just a file containing the word and associated vector),\n",
+    "* or as full models (we will see this in the next notebook).\n",
+    "\n",
+    "Most pretrained models are shared as a vectors-only file, full models are generally kept only if one may update the model at some point."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To upload a file with vectors only, we use the `KeyedVectors` module from `gensim.models`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We need to import KeyedVectors from gensim:\n",
+    "from gensim.models import KeyedVectors"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As an example, we will use a Word2Vec model that was trained on a 4.2-billion-word corpus of 19th-century British newspapers.\n",
+    "\n",
+    "These embeddings were trained to investigate semantic change in the lexicon of mechanization in 19th-century British newspapers, by Nilo Pedrazzini and Barbara McGillivray:\n",
+    "> Pedrazzini, Nilo & Barbara McGillivray. 2022. Diachronic word embeddings from 19th-century British newspapers [Data set]. Zenodo. DOI: https://doi.org/10.5281/zenodo.7181682\n",
+    "\n",
+    "Download the embeddings file from [here](https://zenodo.org/record/7181682). Download just one of the `.txt` files, put it in the `models/` folder.\n",
```

Comment:
maybe use wget instead? (but this might fail when running locally)?

Reply:
Also add mkdir models? Or explicitly mention to create a models folder in Sessions?